code/game/gamemodes/vampire/vampire.dm

```diff
@@ -312,6 +310,17 @@ You are weak to holy things and starlight. Don't go into space and avoid the Cha
 				to_chat(owner, "<span class='notice'><b>You have accumulated [bloodtotal] [bloodtotal > 1 ? "units" : "unit"] of blood[bloodusable != old_bloodusable ? ", and have [bloodusable] left to use" : ""].</b></span>")
 		check_vampire_upgrade()
 		H.blood_volume = max(H.blood_volume - 25, 0)
+		//Blood level warnings (Code 'borrowed' from Fulp)
+		if(H.blood_volume)
+			if(H.blood_volume <= BLOOD_VOLUME_BAD && blood_volume_warning > BLOOD_VOLUME_BAD)
+				to_chat(owner, "<span class='danger'>Your victim's blood volume is dangerously low.</span>")
```

Comment:
Plural of victim is victims, otherwise it reads "Your victim is blood volume is dangerously low"

Reply:
This is the correct grammer. "Victim's" shows ownership of the blood. It is the victim's blood.

Reply:
It might be Victims' actually, but that doesn't look right to me.

Reply:
`Victims'` would imply multiple victims.